lib/response.js

```diff
@@ -116,9 +116,11 @@ Response.prototype.get = function get(name) {
  * @function getHeaders
  * @returns  {Object}
  */
-Response.prototype.getHeaders = function getHeaders() {
-    return (this._headers || {});
-};
+if (typeof Response.prototype.getHeaders !== 'function') {
```

Comment:
Could you please move the JS Doc down into the if block on top of the function? Additionally If you could then add a comment before the if block about why this is necessary, that'd be great. Once we get the comments fixed we should be ready to land this. Thanks so much!